app/components/ui/files/publish-modal/component.js

```diff
@@ -46,30 +50,28 @@ export default Ember.Component.extend({
                     limit: 0,
                     sort: 'lowerName',
                     sortdir: 1
-                }}
+                }};
             self.get('authRequest').send(url) 
             .then(rep => {
                 let fileList2 = []
                 rep.forEach(function(item)
                 {
-                    let path = item.name
-                    fileList2.push({'name': path, 'size':item.size, 'id': item._id})
+                    let path = item.name;
+                    fileList2.push({'name': path, 'size':item.size, 'id': item._id});
                 })
-                self.set('fileList', fileList2)
+                self.set('fileList', fileList2);
             })
         })
-        console.log(self.get('fileList'))
     },
 
     getPath(id) {
-            let url = config.apiUrl + '/item/'+ id + '/rootpath'
+            let url = config.apiUrl + '/item/'+ id + '/rootpath';
             let self = this;
             let path = '';
             let hasRoot = false;
 
             this.get('authRequest').send(url)
                 .then(rep => {
-                    console.log(rep);
                     rep.forEach(function(folder) {
                         let folderName = folder.object['name'];
                         if (folderName == 'Data' | folderName == "Home") {
```

Comment:
this `if` must use `===` and `||` for the _or_ operator, @ThomasThelen 
Sorry I did not catch this earlier

Reply:
And please, use the _hard_ quotation marks (`' '`) consistently and as much as possible, favor them over the _soft_ ones (`" "`)